### PR TITLE
show calculated default value in docs

### PR DESCRIPTION
**Pull request by nnutter**

### CODE REVIEW

**Comment by brummett** [comment]
Does the user really care to know whether the default value is calculated or hardcoded?

File: lib/Command/V1.pm [hunk 1]
```diff
@@ -831,6 +831,11 @@ sub help_options {
             $default_value = "\nDefault value $default_value if not specified";
         }
 
+        my $calculated_default = $property_meta->calculated_default;
+        if (defined $calculated_default) {
```
Comment (ebelter) [inline lib/Command/V1.pm]: What happens if there is a calculated and regular default? Is that allowed in the initializer?

**Comment by ebelter** [comment]
I like that is says it is calculated. Might be confusing, though, and unnecessary to state it.

File: lib/Command/V1.pm (hunk 1 above)
Comment (nnutter) [inline lib/Command/V1.pm]: Good though.  Yeah, the [initializer](https://github.com/genome/UR/blob/aa0b83125df7d8eec1ced630003ff4c21ac610e0/lib/UR/Object/Type/Initializer.pm#L1095-L1097) does block that.

**Comment by nnutter** [comment]
> Does the user really care to know whether the default value is calculated or hardcoded?

I don't have a strong feeling about it but it seemed relevant to me.  I'll defer to you.  Please let me know what you decide.

**Comment by brummett** [comment]
I'd have left out the distinction and made the message for calculated_default the same as default_value - since the class initializer already forbids having both a default_value and calculated_default.

**Comment by nnutter** [comment]
I've pushed a new commit that uses the same message and formatting for `calculated_default` as for `default_value`.

**Comment by ebelter** [comment]
+1

**Comment by brummett** [comment]
+1

**Pull request merged**